core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -194,10 +211,24 @@ Map<String,String> modifyProperties(Consumer<Map<String,String>> mapMutator)
    * @param tserver The tablet server address. This should be of the form
    *        {@code <ip address>:<port>}
    * @return A list of active scans on tablet server.
+   * @deprecated see {@link #getActiveScans(Server)}
    */
+  @Deprecated(since = "3.0.0")
   List<ActiveScan> getActiveScans(String tserver)
       throws AccumuloException, AccumuloSecurityException;
 
+  /**
+   * List the active scans on a tablet server.
+   *
+   * @param server server type and address
+   * @return A collection of active scans on tablet server.
+   * @since 3.0.0
+   * @throws IllegalArgumentException when the type of the server is not TABLET_SERVER or
+   *         SCAN_SERVER
```

Comment:
You could enforce this in the API contract at compile time by having an intermediate ScanningServer type, rather than leave this to a runtime failure.